views/forms/ConfigAssessmentFormFields/index.jsx

```diff
@@ -0,0 +1,77 @@
+import React from 'react'
+import { MenuItem } from '@material-ui/core'
+import ConfigurationCategoryCard from '../../components/ConfigurationCategoryCard'
+import TextInput from '../TextInput'
+import ControlledSelectInput from '../ControlledSelect'
+
+const ConfigAssessmentFormFields = ({
+  classes,
+  control,
+  colors,
+  index,
+  width,
+  onRemove,
+}) => {
+  return (
+    <ConfigurationCategoryCard
+      classes={classes}
+      formIndex={index}
+      onRemove={onRemove}
+      rowNum={index + 1}
+      width={width}
+    >
+      <TextInput
+        control={control}
+        name={`config.${index}.category`}
+        label="Category"
+      />
+      <TextInput
+        control={control}
+        name={`config.${index}.analysis`}
+        label="Assessment"
+      />
+      <TextInput
+        control={control}
+        name={`config.${index}.variable`}
+        label="Variable"
+      />
+      <TextInput
+        control={control}
+        name={`config.${index}.label`}
+        label="Label"
+      />
+      <ControlledSelectInput control={control} name={`config.${index}.color`}>
```

Comment:
This format is important for react hook forms to know which form value state to update.